web/src/composables/useDashboardPanel.ts

```
@@ -893,6 +909,9 @@ const useDashboardPanelData = (pageKey: string = "dashboard") => {
         dashboardPanelData.data.queries[
           dashboardPanelData.layout.currentQueryIndex
         ].config.limit = 0;
+        dashboardPanelData.data.queries[
+          dashboardPanelData.layout.currentQueryIndex
+        ].config.time_shift = [];
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)